src/features/cartographie/layouts/cartographie/cartographie.layout.ts

```diff
@@ -215,4 +215,13 @@ export class CartographieLayout {
   private getRouteParam(routeParam: string) {
     return this.route.children[0]?.children[0]?.snapshot.paramMap.get(routeParam) ?? '';
   }
+
+  public resetFilterOnGeolocate(localisation: Localisation): Localisation {
+    if (localisation)
+      this.router.navigate(['/cartographie'], {
+        queryParams: {},
+        queryParamsHandling: 'merge'
+      });
+    return (this.userLocalisation = localisation);
+  }
```

Comment:
```suggestion
  public resetBreadcrumbOnGeolocate(localisation: Localisation): Localisation {
    if (localisation)
      this.router.navigate(['/cartographie'], {
        queryParams: {},
        queryParamsHandling: 'merge'
      });
    return (this.userLocalisation = localisation);
  }
```
Je te propose ce renommage, car en lisant le nom, j'ai cru que tu réinitialisais les filtres d'orientation, alors qu'il s'agit plutôt du fil d'ariane ([breadcrumb](https://getbootstrap.com/docs/5.3/components/breadcrumb/))